### DISCUSSION TITLE
How to use with WSL2

### DISCUSSION THREAD

**Discussion by jbardnz** [spatie/ray#183] [Q&A]
Hi guys,

I am running WSL2 on Windows 10. I have created a ray.php file and tried a few different IP addresses but can't get it to connect to Ray, any ideas?

**Comment by smoetje** [comment]
Ray works like a breeze on homestead or docker on native Ubuntu. Simple to install and configure.
But with WSL2 in Windows 10 however, I sadly didn't manage to get it working neither (WSL2 running Ubuntu).
